### PR TITLE
update attribute controls for xpress-9.4

### LINKED ISSUE #272: Unrecognized control parameter.
I found parameters `MCFCUTSTRATEGY`, and `HEURSHIFTPROP` are not recognised. Those are new parameters from the Xpress 9.4 release.

### PR DESCRIPTION
**Pull request by mbataillou**
Ran the `build_param_control_dicts.jl` script and kept old parameters for compatibility.

Closes #272

### CODE REVIEW

File: src/attributes_controls.jl [hunk 1]
```diff
@@ -1,27 +1,22 @@
-# Copyright (c) 2016: Joaquim Garcia, and contributors
-#
-# Use of this source code is governed by an MIT-style license that can be found
-# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
-
```
Comment (odow) [inline src/attributes_controls.jl:5]: Please keep

File: src/attributes_controls.jl [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
 Banner from header (xprs.h):
- * (c) Copyright Fair Isaac Corporation 1983-2023. All rights reserved     *
- * For FICO Xpress Optimizer v41.01.03                                     *
+ * (c) Copyright Fair Isaac Corporation 1983-2024. All rights reserved     *
+ * For FICO Xpress Optimizer v43.01.01                                     *
 
 =#
 
-const STRING_CONTROLS = Dict{String,Int32}(
+const STRING_CONTROLS = Dict{String, Int32}(
```
Comment (odow) [inline src/attributes_controls.jl:19]: I think the formatter is going to complain about this

**Review by odow — COMMENTED** [review]
We also need to update the constants at the bottom of https://github.com/jump-dev/Xpress.jl/blob/master/src/Lib/xprs.jl

**Review by mbataillou — COMMENTED** [review]
Added the license back and fixed format.

**Comment by mbataillou** [comment]
I am not sure how to update the constants in  https://github.com/jump-dev/Xpress.jl/blob/master/src/Lib/xprs.jl. Where do I find the new values, the ones there do not seem to map to the attribute controls.

**Comment by mbataillou** [comment]
> Added the license back and fixed format.



> We also need to update the constants at the bottom of https://github.com/jump-dev/Xpress.jl/blob/master/src/Lib/xprs.jl

Updated that in the last commit. Tell me if I missed anything!

**Review by odow — APPROVED** [review]
LGTM

**Pull request merged**